Simulation.cc

```diff
@@ -101,10 +118,10 @@ void setupTrackByToyMC(SVector3& pos, SVector3& mom, SMatrixSym66& covtrk, HitVe
     // can I do what I did, generate one (theta_space) and rotate it by another random numbers
     const float z1 = g_gaus(g_gen);
     const float z2 = g_gaus(g_gen);
-    const float phismear = g_unif(g_gen)*TMath::TwoPi(); // random rotation of scattering plane
+    const float phismear = g_unif(g_gen)*Config::TwoPI; // random rotation of scattering plane
     const float X0 = 9.370; // cm, from http://pdg.lbl.gov/2014/AtomicNuclearProperties/HTML/silicon_Si.html
     //const float X0 = 0.5612; // cm, for Pb
-    float x = 0.1; // cm  -assumes radial impact. This is bigger than what we have in main
+    float x = 0.0025; //.1 cm  -assumes radial impact. This is bigger than what we have in main
```

Comment:
Moved x and x0 to config.  since "x" is everywhere in our code, I made the parameter x previously in Simulation.cc as Config::xr (the r for the radial impact from the comment by Peter)